ee/clickhouse/views/events.py

```diff
@@ -91,13 +91,13 @@ def retrieve(self, request: Request, pk: Optional[int] = None, *args: Any, **kwa
 
     @action(methods=["GET"], detail=False)
     def values(self, request: Request, **kwargs) -> Response:
-
         key = request.GET.get("key")
         team = self.team
         result = []
         if key:
             result = get_property_values_for_key(key, team, value=request.GET.get("value"))
-        return Response([{"name": convert_property_value(value[0])} for value in result])
+            flattened = flatten(result)
+        return Response([{"name": convert_property_value(value[0])} for value in flattened])
```

Comment:
I think this would blow up `if not key`.

Suggestion: `return Response([{"name": convert_property_value(value[0])} for value in flatten(result)])`

Reply:
Fixed